src/BUILD.bazel

```diff
@@ -40,6 +34,15 @@ rollup_bundle(
     deps = ["//src"],
 )
 
+# Needed because the prodserver only loads static files that appear under this
```

Comment:
is this really true? we completely control the args to http-server, is it possible to get it to serve zone.js from a different directory?

Reply:
looks like it is. http-server doesn't support multiple root dirs and couldn't find any other node http server that does